modules/cnv-updating-access-mode-to-enable-live-migration.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-to-enable-live-migration_{context}"]
+= Updating access mode to enable LiveMigration
+
+Storage class defaults in {VirtProductName} are configured to provide ReadWriteOnce (RWO) access mode
+by default. Therefore, LiveMigration is not enabled by default unless you
```

Comment:
LiveMigration is enabled by default iirc. It just requires this accessmode to work.

Reply:
Gotcha. Changed to: 

For LiveMigration to function properly, the storage class must use the 
ReadWriteMany (RWX) access mode.

Storage class defaults in {CNVProductName} are configured to provide 
ReadWriteOnce (RWO) access mode by default. To enable LiveMigration, the 
access mode must be changed to RWX.